libs/tokens/src/updaters/TokensListsUpdater/index.ts

```diff
@@ -0,0 +1,56 @@
+import { useAtom, useAtomValue, useSetAtom } from 'jotai'
+import useSWR, { SWRConfiguration } from 'swr'
+import ms from 'ms.macro'
+import { useEffect } from 'react'
+
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+
+import { allListsSourcesAtom, tokenListsUpdatingAtom } from '../../state/tokenLists/tokenListsStateAtom'
+import { fetchTokenList } from '../../services/fetchTokenList'
+import { environmentAtom } from '../../state/environmentAtom'
+import { getFulfilledResults, getIsTimeToUpdate, LAST_UPDATE_TIME_KEY } from './helpers'
+import { ListState } from '../../types'
+import { upsertListsAtom } from '../../state/tokenLists/tokenListsActionsAtom'
+
+const TOKENS_LISTS_UPDATER_INTERVAL = ms`6h`
```

Comment:
This const is repeated in the previous file

Reply:
Thanks! Fixed